packages/fuel-indexer/src/ffi.rs

```diff
@@ -154,7 +154,16 @@ fn put_object(env: &IndexEnv, type_id: i64, ptr: u32, len: u32) {
         bytes.extend_from_slice(&mem.data_unchecked()[range]);
     }
 
-    let columns: Vec<FtColumn> = bincode::deserialize(&bytes).expect("Serde error.");
```

Comment:
This is where the actual bug is. Not sure what causes 1 out of 178,778 of these to be incorrect but we can look at the data later. For now it just shouldn't crash the service.